StatusWidget/StatusWidget.swift

```diff
@@ -0,0 +1,264 @@
+
+    var body: some View {
+        VStack(alignment: .center, spacing: 5) {
+            HStack(alignment: .center) {
+                LoopCircleView(entry: entry)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                // There is a SwiftUI bug which causes view not to be padded correctly when using .border
+                // Added padding to counteract the width of the border
+                    .padding(.leading, 8)
+                
+                GlucoseView(entry: entry)
+            }
+            .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .center)
+            .padding(5)
+            .background(
+                ContainerRelativeShape()
+                    .fill(Color("WidgetSecondaryBackground"))
+            )
+                        
+            HStack(alignment: .center) {
+                BasalView(entry: entry)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                
+                if let eventualGlucose = entry.eventualGlucose {
+                    let glucoseFormatter = NumberFormatter.glucoseFormatter(for: eventualGlucose.unit)
+                    if let glucoseString = glucoseFormatter.string(from: eventualGlucose.quantity.doubleValue(for: eventualGlucose.unit)) {
+                        VStack {
+                            Text("Eventual")
+                            .font(.footnote)
+                            .foregroundColor(entry.isOld ? Color(UIColor.systemGray3) : Color(UIColor.secondaryLabel))
+                            
+                            Text("\(glucoseString)")
+                                .font(.subheadline)
+                                .fontWeight(.heavy)
+                            
+                            Text(eventualGlucose.unit.shortLocalizedUnitString())
+                                .font(.footnote)
+                                .foregroundColor(entry.isOld ? Color(UIColor.systemGray3) : Color(UIColor.secondaryLabel))
+                        }
+                        .frame(maxWidth: .infinity, alignment: .center)
+                    }
+                }
+            }
+            .frame(maxHeight: .infinity, alignment: .center)
+            .padding(5)
+            .background(
+                ContainerRelativeShape()
+                    .fill(Color("WidgetSecondaryBackground"))
+            )
+        }
+        .foregroundColor(entry.isOld ? Color(UIColor.systemGray3) : nil)
+        .padding(5)
+        .background(Color("WidgetBackground"))
+    }
+}
+
+@main
+struct SmallStatusWidget: Widget {
+    let kind: String = "SmallStatusWidget"
+
+    var body: some WidgetConfiguration {
+        StaticConfiguration(kind: kind, provider: StatusWidgetProvider()) { entry in
+            SmallStatusWidgetEntryView(entry: entry)
+        }
+        .configurationDisplayName("Loop Status Widget")
+        .description("This widget displays your status.")
```

Comment:
This should be more specific. Something like "See your current blood glucose and Loop's dosing status".